### DISCUSSION TITLE
【另一种选择】用同样的理念开发了一款手机App“独白-人人都是多语者”

### DISCUSSION THREAD

**Discussion by sniperboy0829** [ZuodaoTech/everyone-can-use-english#1035] [General]
想要在手机上学习外语的同学，可以在AppStore下载[独白-人人都是多语者](https://apps.apple.com/app/id6648770338)。
“独白”采用了与“人人都能用英语”同样的理念，手机端使用，**无需科学上网**，不需要自己管理Key，轻松学习英语、日语、德语、法语、西班牙语等小语种，有喜欢的同学可以下载下来体验下。https://apps.apple.com/app/id6648770338

**Comment by YoungWingen** [comment]
可惜只支持ios17
